src/controllers/itemdetailpage.js

```diff
@@ -506,15 +508,15 @@ define(["loading", "appRouter", "layoutManager", "connectionManager", "cardBuild
                 index: 0,
                 tag: item.ImageTags.Thumb
             });
-            itemBackdropElement.classList.remove("noBackdrop");
+            page.classList.remove("noBackdrop");
             imageLoader.lazyImage(itemBackdropElement, imgUrl, false);
             hasbackdrop = true;
         } else {
-            itemBackdropElement.classList.add("noBackdrop");
```

Comment:
why?

Reply:
The noBackdrop class is now at the very top of the page, so everything below it has access to it. No need to add it elsewhere.